### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by CaoShaungLi**
Prac 5 code review request @Baicha-S, @Blynxxxx

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
-    #     else:
-    #         print("Invalid short state")
-    #     state_code = input("Enter short state: ").upper()
+    # if state_code in CODE_TO_NAME:
+    #     print(state_code, "is", CODE_TO_NAME[state_code])
+    # else:
+    #     print("Invalid short state")
+    # state_code = input("Enter short state: ").upper()
     try:
-        state_code = state_code in CODE_TO_NAME
```
Comment (Blynxxxx) [inline prac_05/state_names.py]: Should delete this code, I think it is a bug

File: prac_05/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# print(words)
+for word in words:
+    if word in word_to_count:
+        word_to_count[word] += 1
+    else:
+        word_to_count[word] = 1
+# print(word_to_count)
+words = list(word_to_count.keys())
+words.sort()
+maximum_word_length = max(len(word) for word in words)+1
```
Comment (Blynxxxx) [inline prac_05/word_occurrences.py:14]: Why plus 1 for the max word length

Comment (CaoShaungLi) [reply prac_05/word_occurrences.py:14]: It's a typo

File: prac_05/state_names.py (hunk 1 above)
Comment (CaoShaungLi) [reply prac_05/state_names.py]: okie

File: prac_05/word_occurrences.py (hunk 2 above)
Comment (Blynxxxx) [reply prac_05/word_occurrences.py:14]: okie

**Pull request merged**